src/db/migrations/20190715201523_customers-schema.js

```diff
@@ -0,0 +1,8 @@
+
+exports.up = function(knex, Promise) {
+  
+};
+
+exports.down = function(knex, Promise) {
+  
+};
```

Comment:
Empty migration? Le sad 😢

Reply:
I will look into the project and fix it.